clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -626,3 +768,29 @@ def correct_fieldmaps_name(to: PathLike) -> None:
 
     for z in Path(to).rglob("*phasediff_e*_ph*"):
         os.rename(z, z.parent / re.sub(r"phasediff_e[1-9]_ph", "phasediff", z.name))
+
+
+def merge_philips_diffusion(
+    input_file: PathLike,
+    output_file: PathLike,
+    number_of_parts: float,
+    run_num: str,
```

Comment:
Actually, now that I read this again, I don't understand why we need two arguments for input and output files because there is only a single file in the process... This function should only have as argument the path to the JSON file that it modifies:

```python
def merge_philips_diffusion(
    json_file: PathLike,
    number_of_parts: float,
    run_num: str,
) -> None:
```